### DISCUSSION TITLE
Inline latex equations for https://teaching.wence.uk/comp4187/coursework2/

### DISCUSSION THREAD

**Discussion by Julian-Wyatt** [wenceorg/comp4187#80] [Q&A] [answered]
It looks like there is a problem with the inline latex equations for the coursework. 

e.g. The analytical solution is $$ w(x) = \begin{cases} \frac{x(4x^2-3)}{48k}, & \mbox{for } 0 < x < \frac{1}{2} \ \frac{(x-1)(1-8x+4x^2)}{48k}, & \mbox{for } \frac{1}{2} < x < 1 \end{cases} $$

I think it maybe the \begin{cases}?

<img width="771" alt="Screenshot 2022-03-21 at 14 28 32" src="https://user-images.githubusercontent.com/44175667/159282660-2ad86c2d-269d-405d-bae5-b30d80e77f35.png">

**Comment by annereinarz** [comment, marked as answer]
Thanks for pointing it out.

**Reply by wence-** [reply]
Fixed by #82

**Comment by Julian-Wyatt** [comment]
For this analytical solution, should there be some definition for u(x) where x = 0.5, for example maybe:  u(0.5) = 0?

**Reply by annereinarz** [reply]
I'm not sure I fully understand the question, are you asking about the equation u∗(x,y,z)=sin(πx)sin(πy)sin(πz)?

**Reply by Julian-Wyatt** [reply]
Sorry,
For the analytical solution of the Euler Bernoulli beam (Q1), x is defined over 0<x<0.5 and 0.5<x<1.
Is the solution therefore undefined at x=0.5 or does the solution at w(0.5)=0?


Additionally, I don’t believe we’ve covered the Dirac function  δ, I assume, like it says in the question, δ(x− 0.5 )=1 at x=0.5 and 0 everywhere else, which alters the matrix defined by the stencil of the 4th order ODE?

**Comment by annereinarz** [comment]
That makes more sense, thanks for clarifying. The analytical solution w(x) is continuous, which means that it doesn't matter if you approach 0.5 from the left or the right, the value will coincide. I have updated the formula to include 0.5 in the first inequality.
Yes, the Dirac function is only 1 at 0.5 and is zero everywhere else. It should enter in on the right hand side, so you should be able to reuse your calculations for the matrix from the first coursework.

**Reply by Julian-Wyatt** [reply]
Ah that makes sense. When I plotted it, it looks like that was the case - Thanks!

**Reply by Julian-Wyatt** [reply]
I'm not sure if I am missing something, 
I've plotted the analytical solution in desmos (see attached):
<img width="1106" alt="Screenshot 2022-03-25 at 15 07 37" src="https://user-images.githubusercontent.com/44175667/160147367-c4732b2d-a8ad-47f2-941c-1a1c2de83402.png">

Is the second solution missing a minus?
(Which is also confirmed by my estimation of u(x) - in the screen shot I multiplied the second analytical solution by -1 and also multiplied my Dirac function by 10 to get that so I assume I have a constant error somewhere):
<img width="496" alt="Screenshot 2022-03-25 at 15 09 19" src="https://user-images.githubusercontent.com/44175667/160147645-3ebaa9a1-19a5-482a-babf-e15eef136aaa.png">

**Reply by annereinarz** [reply]
Yes, sorry about that. I've added the minus now.
